src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -522,7 +522,7 @@ public sealed partial class JsonIncludeAttribute : System.Text.Json.Serializatio
     {
         public JsonIncludeAttribute() { }
     }
-    [System.AttributeUsageAttribute(System.AttributeTargets.Property, AllowMultiple=false)]
+    [System.AttributeUsageAttribute(System.AttributeTargets.Property | System.AttributeTargets.Field, AllowMultiple=false)]
```

Comment:
`JsonIncludeAttribute` should also have fields as a target. This is helpful when `JsonSerializerOptions.IncludeFields`.

Reply:
Missed this, thanks!